src/backend/oracle-data-api/src/main/resources/occm-openapi-spec.yaml

```diff
@@ -284,6 +284,24 @@ paths:
                 $ref: '#/components/schemas/ResponseResult'
       tags:
         - Violation Ticket
+  /v1/updateViolationTicket:
+    put:
+      requestBody:
+        description: body
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/ViolationTicket'
+        required: true
+      responses:
+        '200':
+          description: Ok
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ResponseResult'
```

Comment:
Just noticed that ReponseResult includes a disputeId and violationTicketId, but these are not used on most returned results. Just thinking that maybe ResponseResult can be a base class and extended with extended properties to return rather than having these properties null most of the time in other scenarios.